### PR TITLE
W-12204799-Add-Kubernetes-service-note-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by seblaz — DISMISSED** [review]
LGTM!

File: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc [hunk 1]
```diff
@@ -262,8 +261,19 @@ spec:
       y-test: false
     host: test.com
 ----
-
-. Enter the following command to view the pods logs and ensure that the API was created successfully: +
++
+. Enter the following command to view the pods logs and ensure that the API is created successfully: +
```
Comment (glenn-rodgers-sf) [inline gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]:
```suggestion
. Enter the following command to view the pods logs and ensure that the API was created successfully: +
```

In my head, this should not be changed. The API was already created when you're looking at this. I think you would only use "is" if you're monitoring the process of it being created.

File: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc [hunk 2]
```diff
@@ -262,8 +261,19 @@ spec:
...
-----
\ No newline at end of file
+----
+
+If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream or you make a request from a policy, you must set:
+
+* `spec.ports[].protocol`, which must be TCP
+* `spec.ports[].name`
+
+Your `spec.ports[].name` service must adhere to the `<protocol>[-<suffix>]` structure. `[-<suffix>]` is optional, and the valid options for `<protocol>` are:
```
Comment (glenn-rodgers-sf) [inline gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]: Is suffix a place holder?

If so, what happens if you don't include it? What protocol is used?

Comment (glenn-rodgers-sf) [reply gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]: Also, is this bullet list related to the above bullet list?

File: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]: OK! Thanks!

File: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]: Yes, the suffix is a placeholder and, as it is optional, it's fine if you don't include it.
Regarding the list, it is part of the same topic -sort of.
Does it make sense, @glenn-rodgers-sf?
Thanks!

File: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc [hunk 3]
```diff
@@ -262,8 +261,19 @@ spec:
...
-
++
 . Enter the following command to view the pods logs and ensure that the API was created successfully: +
 ----
 kubectl logs -f <pod-name>
-----
\ No newline at end of file
+----
+
+If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream or you make a request from a policy, you must set:
```
Comment (glenn-rodgers-sf) [inline gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]:
```suggestion
If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream service or you apply a policy that makes requests to a Kubernetes Service, you must set the following parameters in the `service` resource:
```

File: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc [hunk 4]
```diff
@@ -262,8 +261,19 @@ spec:
...
+
+* `spec.ports[].protocol`, which must be TCP
```
Comment (glenn-rodgers-sf) [inline gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]:
```suggestion
* `spec.ports[].protocol` to `TCP`
```

File: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc (hunk 3 above)
Comment (Crispy-Salesforce) [reply gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]: OK! Thanks!

File: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc (hunk 4 above)
Comment (Crispy-Salesforce) [reply gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc]: OK! Thanks!

**Comment by Crispy-Salesforce** [comment]
Including these changes in the following PR:
W-12204799-Add-Kubernetes-service-note-CP
Closing PR

**Pull request closed without merging**